### PR TITLE
fix: correct typos in docs

### PR DESCRIPTION
**Pull request by vuittont60**
### Detailed summary

- Fixed typos in parameter names in multiple files
- Corrected spelling of "contract" in multiple files
- Updated comments to use "signatures" instead of "singatures"

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Review by avasisht23 — DISMISSED** [review]
Thanks!

File: site/packages/aa-core/utils/getDefaultEntryPointAddress.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ const entryPointAddress = getDefaultEntryPointAddress(chain);
 
 The Address of the default EntryPoint contrafct for the input chain
```
Comment (avasisht23) [inline site/packages/aa-core/utils/getDefaultEntryPointAddress.md:37]: if you could help get these too, would be appreciated!
```suggestion
The Address of the default EntryPoint contract for the input chain
```

File: site/packages/aa-accounts/utils/getDefaultLightAccountFactoryAddress.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ const factoryAddress = getDefaultLightAccountFactoryAddress(chain);
 
 The Address of the default Light Account Factory contrafct address for the input chain
```
Comment (avasisht23) [inline site/packages/aa-accounts/utils/getDefaultLightAccountFactoryAddress.md:37]: if you could help get these too, would be appreciated!
```suggestion
The Address of the default Light Account Factory contract address for the input chain
```

File: site/packages/aa-core/utils/getDefaultSimpleAccountFactoryAddress.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ const factoryAddress = getDefaultSimpleAccountFactoryAddress(chain);
 
 The Address of the default Simple Account Factory contrafct address for the input chain
```
Comment (avasisht23) [inline site/packages/aa-core/utils/getDefaultSimpleAccountFactoryAddress.md:37]: if you could help get these too, would be appreciated!
```suggestion
The Address of the default Simple Account Factory contract address for the input chain
```

**Review by avasisht23 — COMMENTED** [review]
one more request 🙏

**Comment by vuittont60** [comment]
@avasisht23 thanks for your review. I have added commits with your suggestions.

**Review by avasisht23 — APPROVED** [review]

**Comment by moldy530** [comment]
fixed on main

**Pull request closed without merging**